Recorder/OutputType.cpp

```diff
@@ -463,169 +131,15 @@ int to_index(const OutputType config) {
-    if(OutputType::PE11 == L) return "PE";
-    if(OutputType::PE12 == L) return "PE";
-    if(OutputType::PE13 == L) return "PE";
-    if(OutputType::PE22 == L) return "PE";
-    if(OutputType::PE23 == L) return "PE";
-    if(OutputType::PE33 == L) return "PE";
-    if(OutputType::SP == L) return "SP";
-    if(OutputType::SP1 == L) return "SP";
-    if(OutputType::SP2 == L) return "SP";
-    if(OutputType::SP3 == L) return "SP";
-    if(OutputType::EP == L) return "EP";
-    if(OutputType::EP1 == L) return "EP";
-    if(OutputType::EP2 == L) return "EP";
-    if(OutputType::EP3 == L) return "EP";
-    if(OutputType::EEP == L) return "EEP";
-    if(OutputType::EEP1 == L) return "EEP";
-    if(OutputType::EEP2 == L) return "EEP";
-    if(OutputType::EEP3 == L) return "EEP";
-    if(OutputType::PEP == L) return "PEP";
-    if(OutputType::PEP1 == L) return "PEP";
-    if(OutputType::PEP2 == L) return "PEP";
-    if(OutputType::PEP3 == L) return "PEP";
-    if(OutputType::U == L) return "U";
-    if(OutputType::U1 == L) return "U";
-    if(OutputType::U2 == L) return "U";
-    if(OutputType::U3 == L) return "U";
-    if(OutputType::U4 == L) return "U";
-    if(OutputType::U5 == L) return "U";
-    if(OutputType::U6 == L) return "U";
-    if(OutputType::UR1 == L) return "U";
-    if(OutputType::UR2 == L) return "U";
-    if(OutputType::UR3 == L) return "U";
-    if(OutputType::V == L) return "V";
-    if(OutputType::V1 == L) return "V";
-    if(OutputType::V2 == L) return "V";
-    if(OutputType::V3 == L) return "V";
-    if(OutputType::V4 == L) return "V";
-    if(OutputType::V5 == L) return "V";
-    if(OutputType::V6 == L) return "V";
-    if(OutputType::VR1 == L) return "V";
-    if(OutputType::VR2 == L) return "V";
-    if(OutputType::VR3 == L) return "V";
-    if(OutputType::A == L) return "A";
-    if(OutputType::A1 == L) return "A";
-    if(OutputType::A2 == L) return "A";
-    if(OutputType::A3 == L) return "A";
-    if(OutputType::A4 == L) return "A";
-    if(OutputType::A5 == L) return "A";
-    if(OutputType::A6 == L) return "A";
-    if(OutputType::AR1 == L) return "A";
-    if(OutputType::AR2 == L) return "A";
-    if(OutputType::AR3 == L) return "A";
-    if(OutputType::MM == L) return "MM";
-    if(OutputType::MM1 == L) return "MM";
-    if(OutputType::MM2 == L) return "MM";
-    if(OutputType::MM3 == L) return "MM";
-    if(OutputType::MM4 == L) return "MM";
-    if(OutputType::MM5 == L) return "MM";
-    if(OutputType::MM6 == L) return "MM";
-    if(OutputType::MMR1 == L) return "MM";
-    if(OutputType::MMR2 == L) return "MM";
-    if(OutputType::MMR3 == L) return "MM";
-    if(OutputType::RF == L) return "RF";
-    if(OutputType::RF1 == L) return "RF";
-    if(OutputType::RF2 == L) return "RF";
-    if(OutputType::RF3 == L) return "RF";
-    if(OutputType::RF4 == L) return "RF";
-    if(OutputType::RF5 == L) return "RF";
-    if(OutputType::RF6 == L) return "RF";
-    if(OutputType::RM1 == L) return "RF";
```

Comment:
There are special tokens, which are aliases of some other tokens, following a ABAQUS convention.
I do not think a simple regex to strip out digits can serve the original purpose.